qa/automation/nikeev/Send CV.feature

```diff
@@ -0,0 +1,124 @@
+		| CV Attachment          | color highlighting  |
+
+Scenario Outline: User can see a real time validation of 'Name' field
+	Given user filled the Send CV form
+	When user types <words> word(s) with <lenght> <latin> letters and <add> in the 'Name' field
+	Then user sees <notification> notification in red color under the field
+	And the field border line becomes red
+
+	Examples: 
+		| words | lenght       | latin     | add     | notification                           |
+		| 1     | less than 70 | Latin     | null    | Invalid name format                    |
+		| 1     | less than 70 | not Latin | null    | Invalid name format                    |
+		| 2     | less than 70 | not Latin | null    | Invalid name format                    |
+		| 2     | 71 or more   | Latin     | null    | The maximum number of characters is 70 |
+		| 2     | 71 or more   | not Latin | null    | The maximum number of characters is 70 |
+		| 1     | 71 or more   | Latin     | null    | The maximum number of characters is 70 |
+		| 1     | 71 or more   | not Latin | null    | The maximum number of characters is 70 |
+		| 2     | less than 70 | Latin     | numbers | Invalid name format                    |
+		| 2     | less than 70 | Latin     | symbols | Invalid name format                    |
+		| 0     | 0            | null      | null    | This field is required                 |
+
+Scenario Outline: User can see a real time validation of 'Email' field
+	Given user filled the Send CV form
+	When user types '<username><@><lvl2><.><lvl1>' with <lenght> symbols in the 'Email' field
+	Then user sees <notification> notification in red color under the field
+	And the field border line becomes red
+
+	Examples: 
+		| username                              | @    | lvl2                                | .    | lvl1                 | lenght       | notification                           |
+		| a-z,0-9, symbols (except ,<>[]()\)    | null | a-z,0-9,-                           | .    | existing domain      | less than 70 | Invalid email format                   |
+		| a-z,0-9, symbols (except ,<>[]()\)    | @    | a-z,0-9,-                           | null | existing  domain     | less than 70 | Invalid email format                   |
+		| a-z,0-9, symbols (except ,<>[]()\)    | @    | a-z,0-9,-                           | .    | existing  domain     | 71 or more   | The maximum number of characters is 70 |
+		| a-z,0-9, symbols (including ,<>[]()\) | @    | a-z,0-9,-                           | .    | existing  domain     | less than 70 | Invalid email format                   |
+		| a-z,0-9, symbols (except ,<>[]()\)    | @    | a-z,0-9,-                           | .    | non-existent  domain | less than 70 | Invalid email format                   |
+		| a-z,0-9, symbols (except ,<>[]()\)    | @    | a-z,0-9,- (including other symbols) | .    | existing  domain     | less than 70 | Invalid email format                   |
```

Comment:
`a-z,0-9,- (including other symbols)`  Maybe you forgot about multi-level domains (like `<username><@><lvl3><.><lvl2><.><lvl1>`)

Reply:
Yep, I will add a point to allowed characters in 'lvl2'